server/jetstream.go

```diff
@@ -855,7 +878,7 @@ func (a *Account) JetStreamEnabled() bool {
 }
 
 func (jsa *jsAccount) remoteUpdateUsage(sub *subscription, c *client, subject, _ string, msg []byte) {
-	const usageSize = 16
+	const usageSize = 32
```

Comment:
So you see what I was commenting few days ago. If it had been released, you would not be able to add those without having to deal with knowing the servers you are sending the update to (because they would fail to decode).

So maybe as a way to plan for future, the check few lines down should not be `if len(msg) != usageSize` but something like `if len(msg) < usageSize` and then a server will always be able to decode the fields that it knows about. This requires obviously that we don't change size of existing fields.

Reply:
Sounds good. Will update.